corehq/apps/users/models.py

```diff
@@ -173,6 +176,11 @@ def view_web_app(self, master_app_id):
             return True
         return master_app_id in self.view_web_apps_list
 
+    def can_access_app(self, master_app_id):
```

Comment:
Nitpick: I'd just call this `app_id`. We're moving away from using "master" in code, and it's confusing because we have both "master_id" and "master" models in linked apps.

Reply:
Okay. I will update it.